dune

```diff
@@ -0,0 +1 @@
+(dirs :standard \ frontend)
```

Comment:
This allows running a simple `dune build` from the root of the project, without having errors related to something in `frontend/node_modules/foo/bar/deep/deeper/something.ml`